### DISCUSSION TITLE
IndexFailedError: Attempting to index but responded with status "failed"

### DISCUSSION THREAD

**Discussion by growdigital** [tinacms/tinacms#4421] [Help] [answered]
To speed things up with my blog migration, I created some content with my text editor. I then hit this error message when building in Netlify:

```
4:00:17 PM: Starting Tina build
4:00:19 PM: Checking indexing process in Tina Cloud...
4:00:19 PM: IndexFailedError: Attempting to index but responded with status "failed". To retry the indexing process, click the "Reindex" button for "main" in the Tina Cloud configuration for this project.  TinaFetchError: Unable to seed content/english/blog/230705-starting-out.md
4:00:19 PM:     at pollForStatus (/opt/build/repo/node_modules/@tinacms/cli/dist/index.js:2600:15)
4:00:19 PM:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
4:00:19 PM:     at async waitForDB (/opt/build/repo/node_modules/@tinacms/cli/dist/index.js:2621:3)
4:00:19 PM:     at async BuildCommand.execute (/opt/build/repo/node_modules/@tinacms/cli/dist/index.js:2728:7)
4:00:19 PM:     at async BuildCommand.validateAndExecute (/opt/build/repo/node_modules/clipanion/lib/advanced/Command.js:73:26)
4:00:19 PM:     at async Cli.run (/opt/build/repo/node_modules/clipanion/lib/advanced/Cli.js:223:24)
4:00:19 PM:     at async Cli.runExit (/opt/build/repo/node_modules/clipanion/lib/advanced/Cli.js:232:28)
```
(Full deploy log here <https://gist.github.com/growdigital/bc3a6e4db67e2d38245e62fd4f590736>)

I followed the [FAQ instructions here ](https://tina.io/docs/tina-cloud/faq/#how-do-i-resolve-errors-caused-by-unindexed-branches):

> you can force a reindexing of a particular branch by making a whitespace change to the `tina/tina-lock.json` file in that branch, commit the change, and push it to GitHub. This will initiate indexing for the branch and (after a few minutes) the error should be resolved.

However, no joy. I also tried re-indexing via the Project Configuration page:
![Screenshot_2024-01-11_16-10-42](https://github.com/tinacms/tinacms/assets/1061992/21b23dc7-b4ab-4e93-8646-697c9b9e980f)

But still no joy.

![Screenshot_2024-01-11_16-33-42](https://github.com/tinacms/tinacms/assets/1061992/2ffe3c0b-ebc9-4e5c-9b5d-cbe02b7bec02)

Any ideas what I can try next to sort out indexing? Thank you :)

Repository: https://github.com/growdigital/hopegarden.uk

**Comment by kldavis4** [comment]
Run the dev command locally to see if you get an error.

**Reply by growdigital** [reply]
Unfortunately looks like the same error:
```
npm run dev

> bookworm-hugo@3.3.0 dev
> tinacms dev -c "hugo server -D"

Starting Tina Dev Server
(node:70881) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Indexing local files ⠋

Unable to seed content/english/blog/230705-starting-out.md

Unable to start dev server, please fix your Tina config / resolve any errors above and try again
```

**Reply by kldavis4** [reply, marked as answer]
This is a problem with your schema/content. if you add `--verbose` to the dev command you can see the full stack trace:

```
TypeError: Cannot read properties of null (reading 'replace')
    at /Users/kelly/projects/forestry/support/hopegarden.uk/node_modules/.pnpm/@tinacms+graphql@1.4.32_react@18.2.0/node_modules/@tinacms/graphql/dist/index.js:4376:22
    at Array.map (<anonymous>)
    at /Users/kelly/projects/forestry/support/hopegarden.uk/node_modules/.pnpm/@tinacms+graphql@1.4.32_react@18.2.0/node_modules/@tinacms/graphql/dist/index.js:4375:20
    at makeKeyForField (/Users/kelly/projects/forestry/support/hopegarden.uk/node_modules/.pnpm/@tinacms+graphql@1.4.32_react@18.2.0/node_modules/@tinacms/graphql/dist/index.js:3850:33)
    at makeIndexOpsForDocument (/Users/kelly/projects/forestry/support/hopegarden.uk/node_modules/.pnpm/@tinacms+graphql@1.4.32_react@18.2.0/node_modules/@tinacms/graphql/dist/index.js:4349:28)
...
```

I pulled your project down and it looks like the issue is that you have several string list fields defined but they only have an empty list item in them:

```
images: 
- 
captions:
- 
alttext:
- 
```

Either remove the empty list item or put in a value

**Reply by growdigital** [reply]
Fantastic, thank you so much! I've never used typescript before, only frontmatter stuff. Thanks for the `--verbose` tip as well. Will give that a go and report back

**Reply by growdigital** [reply]
My total misunderstanding of TOML, putting in the dashes. I reformatted all the content, rewrote the Hugo archetype, then reinstalled Tina. Seems to have fixed it, thank you :)
